frontend/CoreUI-template/src/views/Article.vue

```diff
@@ -104,7 +174,37 @@ export default {
             console.log("eerror");
           }
         );
+    },
+    postComment() {
+      console.log(this.item.uuid);
```

Comment:
I think we need to remove this line  (ln 179)

Reply:
I updated it. Thanks!